scheduler/src/cook/rest/api.clj

```diff
@@ -1490,7 +1481,7 @@
   cluster, and get back the data for those that do match."
   (let [uuids (wrap-seq (::instances ctx))
         allow-partial-results? (::allow-partial-results? ctx)
-        instance-uuid->job-uuid #(instance-uuid->job-uuid (d/db conn) %)
+        instance-uuid->job-uuid #(util/instance-uuid->job-uuid (d/db conn) %)
```

Comment:
Would it make sense to use the cached call here?

Reply:
(see above comment, same concept)